src/FakeItEasy.Analyzer/UnusedReturnValueAnalyzer.cs

```diff
@@ -57,10 +57,12 @@ private static void AnalyzeCall(SyntaxNodeAnalysisContext context)
 
         private static IMethodSymbol GetCalledMethodSymbol(InvocationExpressionSyntax call, SyntaxNodeAnalysisContext context)
         {
-            var memberAccess = call?.Expression as MemberAccessExpressionSyntax;
-            return memberAccess == null
+            var name = (call?.Expression as MemberAccessExpressionSyntax)?.Name
+                ?? call?.Expression as IdentifierNameSyntax;

```

Comment:
`MemberAccessExpressionSyntax.Name` is a `SimpleNameSyntax`, and `IdentifierNameSyntax` is a subclass of `SimpleNameSyntax`. So `name` is of type `SimpleNameSyntax`.